app/flatpak-builtins-info.c

```diff
@@ -344,7 +344,7 @@ flatpak_builtin_info (int argc, char **argv, GCancellable *cancellable, GError *
       if (opt_show_size)
         {
           maybe_print_space (&first);
-          g_print ("%s", formatted);
+          g_print ("%lu", size);
```

Comment:
Is `guint64` guaranteed to be `unsigned long` on all platforms, or should we use `G_GUINT64_FORMAT`?

Reply:
We could, I guess. Although I'm pretty sure we use %lu in other places, and it probably doesn't matter for the platforms flatpak runs on.

Reply:
`git grep "%lu"` doesn't seem to return anything